src/views/v2/month/calendar-body/day/multiday-events/date/featured.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * View: Month View - Multiday Single Event Featured Icon
```

Comment:
This file should be at `month/calendar-body/day/mutliday-events/multiday-event/date/featured`